mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -85,70 +85,31 @@ Feature: Loans tests
     And match response.itemId == itemId
 
   Scenario: Get Loans collection
-    * def extInstanceTypeId = call uuid1
-    * def extInstitutionId = call uuid1
-    * def extCampusId = call uuid1
-    * def extLibraryId = call uuid1
-    * def extItemId1 = call uuid1
-    * def extItemId2 = call uuid1
-    * def limit = 5
+    
+    * def limit = random(2000)
+    * def extItemBarcode = random(10000)
+    * def extUserBarcode = random(10000)
 
-    #post items
-    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostInstance') { extInstanceTypeId: #(extInstanceTypeId) }
+    #post an item
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostInstance')
     * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostServicePoint')
-    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostLocation') { extInstitutionId: #(extInstitutionId), extCampusId: #(extCampusId), extLibraryId: #(extLibraryId) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostLocation')
     * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostHoldings')
-    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: '989898', extItemId: #(extItemId1) }
-    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: '989899', extItemId: #(extItemId2) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: #(extItemBarcode) }
```

Comment:
to create two items was a good idea, why did you remove one of them I haven't clue